.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,34 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.205.2/containers/javascript-node
+{
+	"name": "Cytorus",
+	"build": {
+		"dockerfile": "Dockerfile",
+		// Update 'VARIANT' to pick a Node version: 16, 14, 12.
+		// Append -bullseye or -buster to pin to an OS version.
+		// Use -bullseye variants on local arm64/Apple Silicon.
+		"args": { "VARIANT": "14-bullseye" }
+	},
+
+	// Set *default* container specific settings.json values on container create.
+	"settings": {},
+
+	// Add the IDs of extensions you want installed when the container is created.
+	"extensions": [
+		"dbaeumer.vscode-eslint",
+		"alexkrechik.cucumberautocomplete"
+	],
+
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	// "forwardPorts": [],
+
+	// Use 'postCreateCommand' to run commands after the container is created.
+	// "postCreateCommand": "yarn install",
```

Comment:
I believe we need to set it to "npm install"

Reply:
I changed it to `npm install` and enabled postCreateCommand.

By the way, if package-lock.json is always kept up to date, I beleve `npm ci` is the best choice. `npm install` may  update package-lock.json.

Nevertheless, the package-lock.json in the current HEAD of master does not contain rewire, which I've added for dev dependency, so `npm install` seems to be the better choice at the moment :wink:

Reply:
I've just updated package-lock.json. If we change to `npm ci`, we'll have to update package-lock.json always for new dependencies separately.

Reply:
I've changed the command to `npm ci`